appformer-client-api/pom.xml

```diff
@@ -35,6 +35,11 @@
       <groupId>javax.enterprise</groupId>
       <artifactId>cdi-api</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.gwt</groupId>
+      <artifactId>gwt-user</artifactId>
+    </dependency>
```

Comment:
I think we could remove `gwt-user` from `appformer-kogito-bridge` as that has dependency for `appformer-client-api` and `mvn dependency:analyze` reports `gwt-user` as unused in `appformer-kogito-bridge`. WDYT?

Reply:
Removed.

Reply:
Ok then I think we need 
```
    <dependency>
      <groupId>com.google.gwt</groupId>
      <artifactId>gwt-user</artifactId>
      <scope>provided</scope>
    </dependency>
```

in `appformer-kogito-bridge` as we import  `appformer-client-api` there. WDYT?

Reply:
Dependencies with scope `provided` are not transitive.